kube/services/amanuensis/amanuensis-deploy.yaml

```diff
@@ -0,0 +1,196 @@
+        readinessProbe:
+          httpGet:
+            path: /_status
+            port: 80
+        ports:
+        - containerPort: 80
+        - containerPort: 443
+        - containerPort: 6567
+        volumeMounts:
+          - name: "logo-volume"
+            readOnly: true
+            mountPath: "/amanuensis/amanuensis/static/img/logo.svg"
+            subPath: "logo.svg"
+          - name: "config-volume"
+            readOnly: true
+            mountPath: "/var/www/amanuensis/amanuensis-config-secret.yaml"
+            subPath: amanuensis-config.yaml
+          - name: "yaml-merge"
+            readOnly: true
+            mountPath: "/var/www/amanuensis/yaml_merge.py"
+            subPath: yaml_merge.py
+        resources:
+            requests:
+              cpu: 0.4
+              memory: 1200Mi
+            limits:
+              cpu: 1.0
+              memory: 2400Mi
+        command: ["/bin/bash"]
+        args:
+          - "-c"
+          - |
+            echo "${AMANUENSIS_PUBLIC_CONFIG:-""}" > "/var/www/amanuensis/amanuensis-config-public.yaml"
+            python /var/www/amanuensis/yaml_merge.py /var/www/amanuensis/amanuensis-config-public.yaml /var/www/amanuensis/amanuensis-config-secret.yaml > /var/www/amanuensis/amanuensis-config.yaml
+            bash /amanuensis/dockerrun.bash && if [[ -f /dockerrun.sh ]]; then /dockerrun.sh; fi
+      - name: nginx-prometheus-exporter-wrapper
+        GEN3_NGINX_PROMETHEUS_EXPORTER_WRAPPER_IMAGE|-image: quay.io/cdis/nginx-prometheus-exporter-wrapper:pybase3-1.4.0-|
+        command: ["/bin/bash"]
+        args:
+          - "-c"
+          - |
+            attempt=0
+            maxAttempts=12
+            while true
+            do
+              nginx_status_response=$(curl -L -s -o /dev/null -w "%{http_code}" -X GET http://localhost/nginx_status)
+              echo "initializing nginx-prometheus-exporter..."
+              echo "nginx_status_response: $nginx_status_response"
+              if [ "$nginx_status_response" == 200 ]; then
+                echo "nginx_status is finally up and running. Proceed with the metrics exporting..."
+                /usr/bin/exporter -nginx.scrape-uri http://127.0.0.1/nginx_status
+              else
+                echo "The nginx_status endpoint is not ready yet... attempt #${attempt}"
+                sleep 5
+                if [ $attempt -eq $maxAttempts ];then
+                  echo "The nginx_status endpoint was never initialized properly, keep the sidecar running in sleep mode to prevent the entire pod from failing..."
+                  sleep infinity
+                fi
+              fi
+              attempt=$(( $attempt + 1 ));
+            done
+        ports:
+        - containerPort: 9113
+      - name: uwsgi-exporter
+        GEN3_UWSGI_EXPORTER_IMAGE|-image: registry.hub.docker.com/timonwong/uwsgi-exporter:v1.0.0-|
+        args: ["--stats.uri", "http://127.0.0.1/uwsgi_status"]
+        ports:
+        - containerPort: 9117
```

Comment:
Not sure what these 2 containers do. I copied the same setting as fence since the backbone of this application is the same as fence. Happy to remove them if they are not necessary